ccmlib/node.py

```diff
@@ -1295,6 +1300,41 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
         except KeyboardInterrupt:
             pass
 
+    @staticmethod
+    def _set_keyspace_initial_tablets(stress_options: List[str], initial_tablets:int = 8,
```

Comment:
Is there some reference to how one should pick this number ?

Reply:
Minimum: 1*(total number of vcpus in the cluster) (suitable for up to 100 tables)
Maximum: 33*(total number of vcpus in the cluster) (suitable for a small number of tables)

For ccm you can treat the boundaries as fairly fluid. Better to have too many than too few. Eventually it will just be a hint and the load balancer will recover from bad inputs.

Reply:
ccm usecase is running scylla on local machines, usually they are not too powerful (and run mutliple scylla nodes) and clusters are not loaded too much with data.
Also, this is just 'initial' number of tablets and in future split/merge will be implemented and this setting will not have that much impact I believe.

I can leave as is currently or implement some logic behind like:
`self._smp * 8` and then to match the nearest bigger power of 2 (IIRC tablets number must be always power of 2):
```
import math

def multiply_and_find_nearest_power_of_two(smp):
     smp *= 8
     smp = math.pow(2, math.ceil(math.log(smp, 2)))
     return smp

```

Reply:
@soyacz where this 8 is coming from ? @avikivity suggested thet the maximum should be used.

Reply:
Note you shouldn't use smp directly, rather the sum of all smp values across the cluster (smp * nodes).

And it doesn't need to be exact.

Reply:
I updated the code with higher number (smp * 32, coerced up to power of 2)